.github/workflows/reusable-build-test-release.yml

```diff
@@ -276,6 +277,12 @@ jobs:
         run: |
           echo "splunk={\"version\":\"unreleased-python3_9-7027496d63d8\", \"build\":\"7027496d63d8\", \"islatest\":false, \"isoldest\":false}" >> "$GITHUB_OUTPUT"
           echo "sc4s={\"version\":\"2.49.5\", \"docker_registry\":\"ghcr.io/splunk/splunk-connect-for-syslog/container2\"}"  >> "$GITHUB_OUTPUT"
+      - name: combined_Splunkmatrix
```

Comment:
Consider removing python39_Splunk and adding logic to combined_Splunkmatrix.
Then all test jobs would require update of matrix and removal of python39 section.

Reply:
Lets postpone those changes for now and focus on delivering fix for modinput tests.